src/main/java/com/github/vkorobkov/jfixtures/loader/FixturesLoader.java

```diff
@@ -29,7 +36,7 @@
     }
 
     private boolean isFile(Path file) {
-        return !file.toFile().isDirectory();
+        return !Files.isDirectory(file);
```

Comment:
I don't get it: I don't use Files.exists in the place, at least, explicitly? I use `java.nio.file.Files#isDirectory` which does not have any warning it jre source/docs